components/Predict/PredictionRequestsTable.jsx

```diff
@@ -1,55 +1,298 @@
 import React, { useEffect, useState } from 'react';
-import { Table } from 'antd/lib';
+import {
+  List, Typography, Spin, Progress, Card, Statistic,
+} from 'antd/lib';
 import dayjs from 'dayjs';
 import { getPredictionRequests } from 'common-util/api/predictionRequests';
 import { useDispatch, useSelector } from 'react-redux';
-import { setPredictionRequests } from 'store/setup/actions';
-
-const columns = [
-  {
-    title: 'Question',
-    dataIndex: 'question',
-    key: 'question',
-  },
-  {
-    title: 'Resolution Time',
-    dataIndex: 'resolution_time',
-    width: 300,
-    key: 'resolution_time',
-    // eslint-disable-next-line camelcase
-    render: (resolution_time) => dayjs.unix(resolution_time).format('HH:mm DD MMM \'YY'),
-  }, {
-    title: 'State',
-    dataIndex: 'state',
-    width: 300,
-    key: 'state',
-    render: (state) => state.charAt(0).toUpperCase() + state.slice(1).toLowerCase(),
-  },
-];
+import {
+  setPredictionRequests,
+  setApprovedRequestsCount,
+} from 'store/setup/actions';
+import { gql } from '@apollo/client';
+import { LoadingOutlined, LinkOutlined } from '@ant-design/icons';
+import PropTypes from 'prop-types';
+import { COLOR } from '@autonolas/frontend-library';
+import client from '../../apolloClient';
+import { ProcessingBanner } from './styles';
+
+const { Text } = Typography;
+
+const GET_FIXED_PRODUCT_MARKET_MAKERS = gql`
+  query GetFixedProductMarketMakers($ids: [ID!]!) {
+    fixedProductMarketMakers(
+      orderBy: creationTimestamp
+      where: { id_in: $ids }
+      orderDirection: desc
+      first: 1000
```

Comment:
This will return 1000 results (maximum allowed). If more results are needed, see how pagination is done here: https://github.com/valory-xyz/trader-quickstart/blob/d0ed1cde6e3116d4ed69e689f8fc7f80f2733bb9/trades.py#L286

Reply:
On both pagination points, we can make improvements when required, thanks